generators/entity-client/templates/react/src/main/webapp/app/entities/entity.reducer.ts.ejs

```diff
@@ -63,8 +63,8 @@ export const ACTION_TYPES = {
 const initialState = {
   loading: false,
   errorMessage: null,
-  entities: [],
-  entity: {},
+  entities: [] as I<%= entityReactName %>[],
+  entity: new <%= entityReactName %>(),
```

Comment:
I'm not so sure about using classes in redux.

In `entity.model.ts.ejs` there currently are both an interface and a class. Is there any situation thinkable where a class would be used, except for setting the default value?

Maybe the class could be replaced with  
```js
// example from travis react-default
export const createDefault = (): IFieldTestEntity => ({
  booleanTom: false,
  booleanRequiredTom: false
});
```

this function could then be used in the initialState

This would also discourage creating methods on the entity model, since that doesn't fit well with react.